tests/unit/backend/corpora/common/entities/test_project.py

```diff
@@ -0,0 +1,38 @@
+import os
+import sys
+import unittest
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "..", "..", "..", "..", ".."))  # noqa
```

Comment:
if we want to run the tests alone it's needed. 

But there is a better way. By adding setting an environment variable `CORPORA_HOME` and pointing to the top level of the project this will be more maintainable. This would become
`pkg_root = os.path.abspath(os.environ["CORPORA_HOME"])

Reply:
@maniarathi Yup as @Bento007 says (thanks for providing the explanation), it's a convention we've used to run standalone tests (i.e. execution dir becomes test file dir as opposed to top level repo dir). I've found it useful in my development workflow so I continue to inject the suspicious code 🙃.

@Bento007 that's a great idea. This pattern is used in multiple files - can we create a ticket to perform this refactor outside of this PR? Granted @maniarathi is on board with keeping the pattern.

Reply:
I'm on board; thanks for the discussion!

Reply:
[Ticket](https://app.zenhub.com/workspaces/corpora-5e2a191dad828d52cc78b028/issues/chanzuckerberg/corpora-data-portal/406). Resolving